MANIFEST.in

```diff
@@ -1,7 +1,7 @@
 # SPDX-FileCopyrightText: Red Hat, Inc.
 # SPDX-License-Identifier: GPL-2.0-or-later
 
-include README.md COPYING MANIFEST
+include README.md LICENSES/* MANIFEST
```

Comment:
All python files use only GPL, so we really need to include only the GPL license.